harmonica/forward/prism.py

```diff
@@ -130,9 +135,26 @@ def prism_gravity(
                 + "mismatch the number of prisms ({})".format(prisms.shape[0])
             )
         _check_prisms(prisms)
+    # Show progress bar for 'jit_prism_gravity' function
+    if progressbar:
+        try:
+            from numba_progress import ProgressBar
+
+            progress_proxy = ProgressBar(total=coordinates[0].size)
+        except ImportError:
+            raise ImportError(
+                "Missing optional dependency 'numba_progress' required if progressbar=True"
+            )
```

Comment:
This works great, but it could be improved.We don't need to try to import `ProgressBar` on every function call, we can move the import step outside the function.

So, I would take the import outside the function, but keeping the `try`, `except` statements to avoid `ImportError` on an optional dependency:

```python
try:
    from numba_progress import ProgressBar
except ImportError:
    ProgressBar = None
```

And inside the function call, we could just raise an `ImportError` if `progressbar` is set to `True` but `ProgressBar` is `None`:

```python
progress_proxy = None
if progressbar:
    if ProgressBar is None:
        raise ImportError("... <custom message> ...")
    progress_proxy = ProgressBar(total=coordinates[0].size)
```

What do you think? Does it sounds better?

Reply:
Yeah, that makes sense, thanks. I've added your suggestion.